docs/layers.rst

```diff
@@ -125,8 +128,45 @@ Finally, it is possible to set the units for the longitude::
 The valid values are ``u.degree`` and ``u.hourangle`` (or simply ``u.hour``) or
 their string equivalents.
 
-Visual attributes
------------------
+Enabling time series attributes
+-------------------------------
+
+If your table has a column of times, you can animate your data by activating 
+the time series attribute and specifying the proper column::
+
+    >>> layer.time_series = True
+    >>> layer.time_att = 'time'
+
+(Please note that time columns must contain 
+`astropy <http://docs.astropy.org/en/stable/time/index.html>`_
+:class:`~astropy.time.Time` objects, :class:`~datetime.datetime` objects,  or 
+`ISOT  
+<https://docs.astropy.org/en/stable/api/astropy.time.TimeISOT.html#astropy.time.
+TimeISOT>`_ compliant strings.)    
+
+Once the time in the viewer matches an object's stated time in the table, its
+corresponding point will pop into view. See :ref:`views` for more information
+on how to control time in the viewer.
+
+By default, time series points disappear a certain amount of time after they 
```

Comment:
This should specify that the default fading time is 15 days (in WWT-time, not real-world time!)

Reply:
I'll specify, though it's 16 days in both the SDK and `layers.py`.